src/assets/data/map_data/professor_offices/BalkanStudies/index.ts

```diff
@@ -2,9 +2,13 @@ import second_floor from "./second_floor";
 import third_floor from "./third_floor";
 import fourth_floor from "./fourth_floor";
 import { DEPARTMENT_CODES } from "../../../DepNames";
-let locations = second_floor.concat(third_floor).concat(fourth_floor);
-let departmentLocations = {
+import { Location } from "../../../../../hooks/useMapData";
+import { DepartmentLocations } from "..";
+const locations: Location[] = [...second_floor,...third_floor, ...fourth_floor];
```

Comment:
you do not need to explicitly type this, its already infered as `Location[]` since all floors are. In general, prefer implicit inference. Code flows better that way